configure

```diff
@@ -7233,7 +7233,6 @@ case $_host_os in
 		# and dead code elimination to reduce binary size
 		if test "$_release_build" = yes; then
 			append_var CXXFLAGS "-flto"
-			append_var CXXFLAGS "-ffunction-sections -fdata-sections"
```

Comment:
These flags also reduce the binary size, by removing unused code and data sections. Are they not needed anymore?